core/src/main/python/wlsdeploy/tool/create/domain_creator.py

```diff
@@ -89,7 +89,8 @@
 from wlsdeploy.tool.util.targeting_types import TargetingType
 from wlsdeploy.tool.util.topology_helper import TopologyHelper
 from wlsdeploy.util import dictionary_utils
-from wlsdeploy.util import model as model_helper
+from wlsdeploy.util import model as model_assist
```

Comment:
To keep this in sync with other imports throughout the code, can this be changed to 
"import wlsdeploy.util import model" without the "as"?  All of the other imports for model and almost all of the other wlsdeploy.util imports are written that way.

Reply:
I considered, but I figured the original author did not want to name it model for confusion. Just a consideraton